packages-new/components/package.json

```diff
@@ -27,10 +27,13 @@
     "@ovhcloud/ods-component-popover": "^15.0.1",
     "@ovhcloud/ods-component-progress-bar": "^15.0.1",
     "@ovhcloud/ods-component-quantity": "^15.0.1",
+    "@ovhcloud/ods-component-radio": "^15.0.1",
```

Comment:
Forget this in radio & search-bar.
Should I create new PR ?

Reply:
no it's ok if you correct this here directly